### PR TITLE
Add TCK cases about NULL comparison and NULL with math functions

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

**Comment by jievince** [comment]
Well done! Could you format the tck tests by `make fmt` in the directory `nebula-graph/tests` at first?

**Comment by whitewum** [comment]
These cases are shown in the doc. So if any feature breaks these cases, you will notice.

**Comment by yixinglu** [comment]
before formating your feature file, you can follow [the tutorial](https://github.com/vesoft-inc/nebula-graph/blob/master/tests/README.md#init-environment) to initialize your environment.

**Review by yixinglu — APPROVED** [review]

File: tests/tck/features/expression/Null.feature [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
```
Comment (laura-ding) [inline tests/tck/features/expression/Null.feature:1]: 2021

**Review by laura-ding — APPROVED** [review]

**Pull request merged**